loadtest/control/actions.go

```diff
@@ -345,6 +360,8 @@ func CreateDirectChannel(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("direct channel for user %v created, id %v", u.Store().Id(), channelId)}
 }
 
+// ViewChannel performs a view action in a random team/channel for the given
+// user. Which will mark all posts as read in the channel.
```

Comment:
Sorry I meant adding a fullstop here stops the flow of the sentence and makes it grammatically incorrect. A comma is fine if you want to give a pause, otherwise just remove the period. If you want to give a period, then the next sentence should start with "This will mark all .. "

Reply:
Oh, I see now :) fixed.